resource/airflow_dag_pipeline.yaml

```diff
@@ -98,59 +113,35 @@ jobs:
           - name: image
 
   - aggregate:
-    - task: update-to-test-iam-role-name
+    - task: -list-policy
       config:
         platform: linux
         image_resource:
           type: docker-image
           source:
-            repository: concourse/docker-image-resource
-        inputs:
-          - name: pull-request
+            repository: quay.io/s_block/alpine-git-jq
         run:
-          path: sh
+          path: bash
           args:
             - -ecx
             - |
-              cp -R ./pull-request/* ./pull-request-edit
-              jq '.role_name = "airflow_test_dag_role"' ./pull-request/deploy.json > ./pull-request-edit/deploy.json
-              jq '.Statement += [{"Sid": "denyall","Action": ["*"],"Effect": "Deny","Resource": ["*"]}]' ./pull-request/iam_policy.json > ./pull-request-edit/iam_policy.json
-              cat ./pull-request-edit/deploy.json
+              echo '[]' | jq -S . > ./policies/list.json
```

Comment:
Are we appending to an input here or creating a new file?

If we're just initialising this to "empty array", we could hardcode that in the `policies=` command instead of `cat`-ing this file 🤷‍♂

Reply:
echoing vars seems to mess with the formatting and didn't work. Writing to the file and cat seemed to be the only way to make it work. But as Andy said, I might do this in python as well.